src/flint/FlintTypes.jl

```diff
@@ -67,6 +67,8 @@ mutable struct fmpz <: RingElem
     fmpz(x::fmpz) = x
 end
 
+const IntegerUnion = Union{Integer,fmpz}
```

Comment:
Should there be a docstring here? If so, what should it say?

Reply:
Of course this doesn't include Singular.ZZ, which is unfortunate. I don't see a way to do that cleanly, other than to have a union there also which includes this one, which is called something else (maybe just Singular.IntegerUnion not sure).

Anyhow, here it is enough to simply add something to the integers.md file at the top, stating that we provide a union of all Julia and Nemo integers called IntegerUnion.

Reply:
OK, I've added a little blurb, let me know what you think

Reply:
Looks good. Thanks.